tests/unit/test_estimator.py

```diff
@@ -3319,7 +3317,6 @@ def test_register_inference_image(sagemaker_session):
     framework = "TENSORFLOW"
     framework_version = "2.9"
     nearest_model_name = "resnet50"
-    data_input_config = '{"input_1":[1,224,224,3]}'
```

Comment:
Have atleast 1 new test where we check for data_input_config if not already present.

Reply:
It is already present in the existing test files.